client/src/index.jsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import App from './components/App.jsx';
+import App from './components/App';
```

Comment:
Yesterday we saw how adding '.jsx' to the end of the file name helped fix a couple of issues. Just want to make sure that this is still the case or if you're good to go without using '.jsx' at the end.

Reply:
I was able to modify my webpack.config.js file so I could use extensions: ['*', '.js', '.jsx']